app/src/main/java/mozilla/lockbox/presenter/ItemDetailPresenter.kt

```diff
@@ -16,13 +18,32 @@ import mozilla.lockbox.store.DataStore
 interface ItemDetailView {
     var itemId: String?
     fun updateItem(item: ItemDetailViewModel)
+
+    val hostnameClicks: Observable<Unit>
 }
 
 class ItemDetailPresenter(
     private val view: ItemDetailView,
     private val dispatcher: Dispatcher = Dispatcher.shared,
     private val dataStore: DataStore = DataStore.shared
 ) : Presenter() {
+
+    override fun onViewReady() {
+        this.view.hostnameClicks
+                .subscribe {
+                    view.itemId?.let {
+                        dataStore.get(it)
+                                .subscribe {
+                                    if (it != null) {
+                                        dispatcher.dispatch(RouteAction.OpenWebsite(it.formSubmitURL!!))
```

Comment:
Please check that this should be the `formSubmitURL`. I have a feeling it should be the `hostname`. cc @linuxwolf ?

Reply:
Technically this should be `hostname`.  In practice I don't _think_ `hostname` and `formSubmitURL` are different, but in theory there's very little in Firefox to encourage/enforce that.

Reply:
updated with hostname